S/Singular/build_tarballs.jl

```diff
@@ -50,15 +49,15 @@ products = [
     # LibraryProduct("interval", :interval),
     LibraryProduct("libfactory", :libfactory),
     LibraryProduct("libsingular_resources", :libsingular_resources),
-    LibraryProduct("libomalloc", :libomalloc)
+    LibraryProduct("libomalloc", :libomalloc),
 ]
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("FLINT_jll"),
+    Dependency("cddlib_jll"),
+    Dependency(PackageSpec(name="FLINT_jll", version=v"2.6.2")),
```

Comment:
This means that the newest `Singular_jll` version has a compat entry for `FLINT_jll`, which requires **exactly** version v2.6.2 of `FLINT_jll`. This means that it won't work with the other packages that need 2.6.3.

Reply:
Can you please submit a rev then which fixes this? Thanks!

Reply:
If we change nothing else, then we would have to do it directly in the registry, see https://github.com/JuliaPackaging/Yggdrasil/pull/1447

On other hand, we may have to do a new version of Singular with fixes for the bugs we found.

Reply:
Either or both sound good to me